packages/core-data/src/test/selectors.js

```diff
@@ -9,10 +9,13 @@ import deepFreeze from 'deep-freeze';
 import { getTerms, isRequestingCategories, getEntityRecord, getEntityRecords } from '../selectors';
 import { select } from '@wordpress/data';
 
-jest.mock( '@wordpress/data', () => ( {
-	...require.requireActual( '@wordpress/data' ),
-	select: jest.fn().mockReturnValue( {} ),
-} ) );
+jest.mock( '@wordpress/data', () => {
+	return {
+		select: jest.fn().mockReturnValue( {
```

Comment:
Did we mean to drop the `...require.requireActual` ?

Reply:
I think I commented elsewhere, yes, it was failing when using `requireActual` because of some Babel + Enzyme internal things. `requireActual` isn't going through `moduleNameMapper` flow so it tried to load Babel build, but couldn't import it properly 🤷‍♂️